README.md

```diff
@@ -1,96 +1,68 @@
-Overleaf
-========
+<h1 align="center">
+  <br>
+  <a href="https://www.overleaf.com"><img src="https://images.ctfassets.net/nrgyaltdicpt/6gsvc5Ogjmu04I4Miu0uGg/cb1d4391717d2ab8d5e42ede6fb0eef1/overleaf_wide_colour_light_bg.png" alt="Overleaf" width="300"></a>
```

Comment:
Would https://www.overleaf.com/img/ol-brand/logo-horizontal.png work? I see that this is the contentful CDN, not sure we should be hotlinking to external assets.

Reply:
my fault, looking at `ctfassets` I took for granted it was a specific DNS service. I'll upload the images to Github, it's the best option.

Reply:
I have also been wondering about this... At the moment on the https://www.overleaf.com/for/partners/logos page we're telling people to link to assets on the CDN, so we're kind of committed to keeping them stable now. So... not sure.

Reply:
We've been talking about retiring CDN77 and move to cloudflare, and I'm not sure that cdn.overleaf.com would actually be needed if we did that. Anyway... we control this, so it's not a big deal where it links to. As Miguel says, put the files on GitHub seems like a good solution (keeps it all in one place).

Reply:
I was more thinking of our guidelines on `/for/partners/logos`

> we control this, so it's not a big deal where it links to

Right, but not necessarily if contentful changes their implementation (I think we get these urls from the contentful api, so it in theory could change.

> the files on GitHub seems like a good solution

Agreed

Reply:
It makes sense to unify the CDN solutions, specially if we're considering moving fonts out of Google Fonts and delivering FontAwesome ourselves.!